### DISCUSSION TITLE
After disable copilot globally, unable to enable it again

### DISCUSSION THREAD

**Discussion by jeremygliksman** [community/community#42196] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Sometimes I turn off copilot because I am still studying the basics of p5js and javascript, and I don't want help from the AI. But for now, I do want it. I tried to enable it globally, but it remains disabled. When I go to the extension page, it says subscription required.

I then visited my GitHub student account and saw that I could use GitHub for free (which I already knew, obviously).

After uninstalling and reinstalling the extension, nothing changed. The installation manual says I should be prompted for a login or an authorisation; I'm receiving neither.

Not sure what I can do more...

**Comment by Akash1134** [comment]
Hi @jeremygliksman ,

Sorry that you're not able to re-enable Copilot. Which IDE are you using it with? Please check out this similar [discussion](https://github.com/orgs/community/discussions/41076) and let us know if that helps. Thanks!

**Reply by jeremygliksman** [reply]
Hi @Akash1134 , thank you for the reply. Unfortunately, I don't have the same issue as the topic creator of the URL you sent me.
I tried to follow the instruction from the URL you sent me, but I don't even have 'tools' in my menu

I am using the latest VSCode app.

Usually, I dis/enable Copilot from the icon in the bottom right corner of VSCode. When I click it, I get the option to 'enable globally.' I can press it, but nothing happens.... Then when I open the extension tab, it says 'Subscription Required'. I press it; it brings me to the login page of GitHub; I log in, and I restart my VSCode but nada...

**Reply by jeremygliksman** [reply]
@Akash1134  Do you maybe have another solution?

**Comment by jfullstackdev** [comment]
I'm not very sure of the specific setup, but do you have any chance of logging out then log in again ? and if it is possible in all the apps you are using with this Copilot

**Reply by jeremygliksman** [reply]
I logged in and out numerous times.
I reinstalled the extension numerous times.

Nothing works :s

I don't use anything more than Copilot. I don't understand much of Github in general because 3 months ago, I never did anything related to development and coding.
